packages/web-api-send-notification-job-manager/src/task/send-notification-task-property-provider.spec.ts

```diff
@@ -23,7 +23,7 @@ describe(SendNotificationTaskPropertyProvider, () => {
         environmentSettingsMock.setup(s => s.getValue('APPINSIGHTS_INSTRUMENTATIONKEY')).returns(() => appInsightsKey);
         environmentSettingsMock.setup(s => s.getValue('KEY_VAULT_URL')).returns(() => keyVaultUrl);
         environmentSettingsMock.setup(s => s.getValue('RUNNER_SCRIPTS_CONTAINER_NAME')).returns(() => runnerScriptContainerName);
-        environmentSettingsMock.setup(s => s.getValue('AZURE_STORAGE_SCAN_QUEUE')).returns(() => 'notification-queue-name');
+        environmentSettingsMock.setup(s => s.getValue('AZURE_STORAGE_NOTIFICATION_QUEUE')).returns(() => 'notification-queue-name');
```

Comment:
we don't need this since you removed the environment settings entry in product code

Reply:
will remove it in the next pr